Hides my sins

juxjux9930 opened the pull request:
after several months of avoiding my shame, this is me confronting my bad sprite with an equally bad sprite
run, cowards, for with my self confidence I am unstoppable.

🆑 Jux
rscadd: Replaces the unused CTI voidsuit with a new one, for nerds.
rscadd: The new Retro Voidsuit can now spawn in the random spawners.
/🆑
![done2](https://user-images.githubusercontent.com/68120725/137221643-23bb5456-6dd3-40ee-8492-dc18047d47ac.PNG)



I'm trying to learn by looking at Azlan and Pawn's voidsuit sprites. There's probably some faults.

SierraKomodo commented on code/game/machinery/suit_storage_units.dm:
```diff
@@ -108,12 +108,12 @@
-	islocked = 1
\ No newline at end of file
+	req_access = null
+	islocked = 0
```
```suggestion
	islocked = FALSE
```

SierraKomodo commented on code/game/machinery/suit_storage_units.dm:
```diff
@@ -108,12 +108,12 @@
+/obj/machinery/suit_storage_unit/nerd
+	name = "Retro Voidsuit Storage Unit"
+	suit= /obj/item/clothing/suit/space/void/nerd
+	helmet = /obj/item/clothing/head/helmet/space/void/nerd
```
I'd prefer the type path to relatively match the actual name. Makes it easier to find in `spawn` lookups.

```suggestion
/obj/machinery/suit_storage_unit/retro
	name = "Retro Voidsuit Storage Unit"
	suit= /obj/item/clothing/suit/space/void/retro
	helmet = /obj/item/clothing/head/helmet/space/void/retro
```

SierraKomodo commented on code/game/machinery/suit_storage_units.dm:
```diff
@@ -108,12 +108,12 @@
-	req_access = list(access_xenoarch)
-	islocked = 1
\ No newline at end of file
+	req_access = null
```
The inherited value from the parent type is already blank.

```suggestion
```

SierraKomodo commented on code/modules/clothing/spacesuits/void/station.dm:
```diff
@@ -453,34 +453,34 @@
+/obj/item/clothing/suit/space/void/nerd
+	name = "Retro Voidsuit"
+	desc = "A voidsuit designed to replicate older designs with newer technology."
+	icon_state = "rig-nerd"
```
Same request for the item state names if you could.
```suggestion
	icon_state = "rig-retro"
```

SierraKomodo commented on code/modules/clothing/spacesuits/void/station.dm:
```diff
@@ -453,34 +453,34 @@
-	icon_state = "rig-ceti"
+//Retro
+/obj/item/clothing/suit/space/void/retro
+	name = "Retro Voidsuit"
```
Didn't realize this was going to be silly until I tried to put it on. Capitalization isn't necessary here and causes funny shenanigans with `\the` (And I'm unsure why the CTI voidsuit had Voidsuit capitalized either)

```suggestion
	name = "retro voidsuit"
```

SierraKomodo commented on code/modules/clothing/spacesuits/void/station.dm:
```diff
@@ -453,34 +453,34 @@
-	icon_state = "rig0-ceti"
-	item_state = "helm-ceti"
+/obj/item/clothing/head/helmet/space/void/retro
+	name = "Retro voidsuit helmet"
```
```suggestion
	name = "retro voidsuit helmet"
```

juxjux9930 committed:
Replaces the Ceti Voidsuit


Update code/game/machinery/suit_storage_units.dm

Co-authored-by: Sierra Brown <SierraKomodo@users.noreply.github.com>
Update code/game/machinery/suit_storage_units.dm

Co-authored-by: Sierra Brown <SierraKomodo@users.noreply.github.com>
Sierra is a naming buzzkill


Update code/modules/clothing/spacesuits/void/station.dm

Co-authored-by: Sierra Brown <SierraKomodo@users.noreply.github.com>
Update code/modules/clothing/spacesuits/void/station.dm

Co-authored-by: Sierra Brown <SierraKomodo@users.noreply.github.com>
